qlib/utils/resam.py

```diff
@@ -8,7 +8,9 @@
 from .time import Freq, cal_sam_minute
 
 
-def resam_calendar(calendar_raw: np.ndarray, freq_raw: Union[str, Freq], freq_sam: Union[str, Freq]) -> np.ndarray:
+def resam_calendar(
+    calendar_raw: np.ndarray, freq_raw: Union[str, Freq], freq_sam: Union[str, Freq], region: str
```

Comment:
I think we can make the region an optional parameter.
And the default value is C['region'] if it is None

Otherwise, all the places need explicitly values when call this function.
![image](https://user-images.githubusercontent.com/465606/163558732-a4032f90-0cf8-4d9e-904f-c43da2e55b3b.png)